tools/cmake/toolchain/openwrt.cmake

```diff
@@ -48,5 +48,7 @@ set(CMAKE_C_IMPLICIT_INCLUDE_DIRECTORIES ${PLATFORM_TOOLCHAIN_DIR} CACHE STRING
 set(CMAKE_CXX_IMPLICIT_INCLUDE_DIRECTORIES ${PLATFORM_TOOLCHAIN_DIR} CACHE STRING "")
 
 ## Toolchain Programs
+set(CMAKE_AR ${PLATFORM_TOOLCHAIN_DIR}/bin/${PLATFORM_TOOLCHAIN_PREFIX}ar CACHE STRING "Archiver")
```

Comment:
I would explain this a bit more in the commit message - this is NOT used right now, but once we switch to building statically we will need this for cross-compile.